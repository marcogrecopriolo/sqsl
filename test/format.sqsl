{ format type  }

display 'display 0, 1.1, "a|" format brief';
display 0, 1.1, "a|" format brief;

display 'display 0, 1.1, "a|" format delimited';
display 0, 1.1, "a|" format delimited;

display 'display 0, 1.1, "a|" format full';
display 0, 1.1, "a|" format full;

display 'display 0, 1.1, "a|" format vertical';
display 0, 1.1, "a|" format vertical;

{ delimited }

display 'display 0, 1.1, "a|" format delimited by ","';
display 0, 1.1, "a|" format delimited by ",";

display 'display 0, 1.1, "a|" format delimited by "|" escape ">"';
display 0, 1.1, "a|" format delimited by "|" escape ">";

{ full }

display 'display 0, 1.1, "a|" format full "%04i %6.3f %10s"';
display 0, 1.1, "a|" format full "%04i %6.3f %10s";

display 'display 0, 1.1, "a|" format full "%-4i %6.3e %-10s"';
display 0, 1.1, "a|" format full "%-4i %6.3e %-10s";

display 'display 0, 1.1, "a|" format full "[&&&&] [###.##] b%sb"';
display 0, 1.1, "a|" format full "[&&&&] [###.##] b%sb";

