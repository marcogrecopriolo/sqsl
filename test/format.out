display 0, 1.1, "a|" format brief
0,1.1,a|
display 0, 1.1, "a|" format delimited
0|1.1|a\||
display 0, 1.1, "a|" format full
0 1.1 a|
display 0, 1.1, "a|" format vertical
 0
 1.1
 a|

display 0, 1.1, "a|" format delimited by ","
0,1.1,a|,
display 0, 1.1, "a|" format delimited by "|" escape ">"
0|1.1|a>||
display 0, 1.1, "a|" format full "%04i %6.3f %10s"
0000  1.100         a|
display 0, 1.1, "a|" format full "%-4i %6.3e %-10s"
0    1.100e+00 a|        
display 0, 1.1, "a|" format full "[&&&&] [###.##] b%sb"
0000   1.10 ba|b
